src/compiler/index.ts

```diff
@@ -1,4 +1,4 @@
 import './polyfills';
 import {compile} from './compile';
 
-(globalThis as any)['compile'] = compile;
+globalThis.compile = compile;
```

Comment:
WDYT about moving the `globalThis` assignments into one `globals.ts` file so we're not as dependent on side-effects in imports? (perhaps not in this PR)

Reply:
I'd argue this isn't an import...its an entrypoint. The only import where we depend on side effects is polyfills, but that is also a bindingless import and it is expected

Reply:
SGTM